scripts/pull_github_pr.sh

```diff
@@ -25,7 +25,9 @@ NL=$'\n'
 
 PR_NUM=$1
 # convert full repo URL to its project/repo part:
-PROJECT=`git config --get remote.origin.url | sed 's/git@github.com://;s#https://github.com/##;s/\.git$//;'`
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+REMOTE="$(git config --get "remote.$BRANCH.url")"
```

Comment:
This, "git config --get "remote.next.url", doesn't seem to return anything on my setup. Something seems wrong here, but I don't know to suggest what should be the right command.

Reply:
It's wrong, I'll fix it. Don't know how it passed my testing.